src/particleContainer/adapter/RDFCellProcessor.cpp

```diff
@@ -33,6 +33,13 @@ void RDFCellProcessor::processCell(ParticleCell& cell) {
 
 				if (dd < _cutoffRadiusSquare) {
 					_rdf->observeRDF(molecule1, molecule2, dd);
+					if (_rdf->doARDF() == true) {
```

Comment:
```suggestion
					if (_rdf->doARDF()) {
```
simplify this statement